code/game/objects/items/mountable_frames/trophymount.dm

```diff
@@ -0,0 +1,111 @@
+/obj/item/mounted/frame/trophy_mount
+	name = "trophy mount"
+	desc = "Used to hang your old hunting trophies. "
+	icon = 'icons/obj/objects.dmi'
+	icon_state = "wallmount"
+	anchored = 0
+	var/initial_name = "wooden wall mount"
+	var/initial_desc = "Used to hang your old hunting trophies. "
+	var/has_trophy = 0
+	var/obj/item/weapon/trophy/trophy = null
+	var/trophy_id
+
+/obj/item/mounted/frame/trophy_mount/do_build(turf/on_wall, mob/user)
+	to_chat(user, "<span class='notice'>You begin attaching the mount to the wall...</span>")
+	if(do_after(user, 50, target = src))
+		to_chat(user, "<span class='notice'>You attach the mount to the wall.</span>")
+		var/obj/item/mounted/frame/trophy_mount/M = new /obj/item/mounted/frame/trophy_mount(get_turf(src), get_dir(on_wall, user), 1)
+		M.pixel_y -= (loc.y - on_wall.y) * 32
+		M.pixel_x -= (loc.x - on_wall.x) * 32
+		M.anchored = 1
+		qdel(src)
+
+/obj/item/mounted/frame/trophy_mount/attack_hand(mob/user) //Dumb, I know
+	if(anchored)
+		return
+	else
+		..()
+
+/obj/item/mounted/frame/trophy_mount/attackby(obj/item/weapon/W, mob/user)
+	if(istype(W, /obj/item/weapon/trophy))
+		if(anchored)
+			var/obj/item/weapon/trophy/T = W //kill me
+			if(!has_trophy)
+				icon_state = "wallmount_[T.id]"
+				name = "[T.id] trophy head"
+				desc += "It has a [T.id] trophy head attached to it."
+				has_trophy = 1
+				trophy_id = T.id
+				trophy = T
+				trophy.loc = src
+				qdel(T)
```

Comment:
I'm guessing you never tested this, because this quite literally adds it to the frame then immediately deletes it.